app/src/main/java/gc/david/dfm/feedback/FeedbackPresenter.java

```diff
@@ -0,0 +1,65 @@
+package gc.david.dfm.feedback;
+
+import android.content.ActivityNotFoundException;
+import android.content.Intent;
+import android.net.Uri;
+
+import gc.david.dfm.DeviceInfo;
+import gc.david.dfm.PackageManager;
+import gc.david.dfm.R;
+import gc.david.dfm.logger.DFMLogger;
+
+/**
+ * Created by david on 07.12.16.
+ */
+public class FeedbackPresenter implements Feedback.Presenter {
```

Comment:
Not totally convinced of this presenter class, maybe a _util_ class method is enough.